tests/test_transforms/test_math/test_statistics_transform.py

```diff
@@ -39,6 +54,8 @@ def simple_df_for_agg() -> pd.DataFrame:
         (MeanTransform, "test_mean"),
         (StdTransform, None),
         (StdTransform, "test_std"),
+        (MADTransform, None),
+        (MADTransform, "test_mad"),
     ),
 )
 def test_interface_simple(simple_df_for_agg: pd.DataFrame, class_name: Any, out_column: str):
```

Comment:
May be it is better to add docstring with description to the test like "Test that everything is fine"(oversimplified).

Reply:
@iKintosh says no comments for tests 😄